dev-infrastructure/templates/aks-development.bicep

```diff
@@ -25,33 +28,14 @@ param osDiskSizeGB int = 32
 param maxPods int = 100
 
 @description('The version of Kubernetes.')
-param kubernetesVersion string = '1.27.7'
-
-@description('Network plugin used for building Kubernetes network.')
-@allowed([
-  'azure'
-  'kubenet'
-])
-param networkPlugin string = 'azure'
-
-@description('boolean flag to turn on and off of RBAC')
-param enableRBAC bool = true
-
-@description('Name of the VNET that will contain the AKS cluster and related resources.')
-param vnetName string = 'aks-net'
+param kubernetesVersion string = '1.28.5'
```

Comment:
should this match the default we have in `standalone` 

`AKSVERSION=1.29.2`

Reply:
Yeah, actually I think we can even just take out the default value here and rely on parameter files